techniques/failures/F102.html

```diff
@@ -12,115 +11,85 @@
       </h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies.</p>
-            <p>This technique relates to:</p>
-            <ul>
-               
-               <li>Success Criterion 1.4.10: Reflow (Failure)</li>
-               
-            </ul>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/reflow">Success Criterion 1.4.10: Reflow</a> (Failure)</span>.
+            <h2>適用 (対象)</h2>
+            <p>全てのウェブコンテンツ技術</p>
+            <p>この達成方法は、<span><a href="https://waic.jp/docs/WCAG21/Understanding/reflow">達成方法 1.4.10: リフロー</a> (失敗)</span> に関連する。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>This document describes a failure that occurs when a change of the viewport width
-               to 320px makes content disappear that was available at wider viewport widths. Some
-               content available at wider widths may not be shown in the same way or at the same
-               position at the viewport width of 320px, simply because there is less space (screen
-               'real estate') to display it. This content, however, should still be available after
-               reflow to 320px viewport width, either by being repositioned in a single column view,
-               or through some interaction offering the information in some other way, for example,
-               in a disclosure area, a dialog, or via a link to another view.
+            <h2>解説</h2>
+            <p>この文書は、ビューポート幅を 320 ピクセルに変更したときに生じる、より広いビューポート幅で利用可能だったコンテンツが表示されなくなる不具合について記述するものである。より広いビューポート幅で利用可能なコンテンツの中には、ビューポート幅が 320 ピクセルになると、単純に表示できるスペース (画面上の「土地」) が少なくなるため、同じような形で又は同じ位置で表示されないものがある。しかし、このようなコンテンツは、320 ピクセルのビューポート幅にリフローされた後であっても、単一のカラムビューに再配置されること、あるいは、情報展開エリア、ダイアログ、又は他のビューへのリンクなど他の方法で情報を提供する何らかのインタラクションによって、利用可能であるべきである。
```

Comment:
最初の文は、whenの取る位置を修正しています。
最後の文は、for example以下のorを単なる読点とし、either ～,or ... をより明示してみました。

```suggestion
            <p>この文書は、ビューポート幅を 320 ピクセルに変更すると、より広いビューポート幅で利用可能だったコンテンツが表示されなくなるときに生じる失敗について説明するものである。より広いビューポート幅で利用可能なコンテンツの中には、ビューポート幅が 320 ピクセルになると、単純に表示できるスペース (画面上の「土地」) が少なくなるため、同じような形で又は同じ位置で表示されないものがある。しかし、このようなコンテンツは、320 ピクセルのビューポート幅にリフローされた後であっても、単一のカラムビューに再配置されることによって、又は、情報展開エリア、ダイアログ、他のビューへのリンクなど他の方法で情報を提供する何らかのインタラクションによってのいずれかで、利用可能であるべきである。
```